### ISSUE TITLE
QA Ph3_Accessibility - Mobile: Schedule - Cannot select "add to calendar" buttons with talk back enabled

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#673]
With talk back enabled, the "add to calendar" buttons on the calendar pages cannot be selected. 

Steps: 
1. Enable talk back in device accessibility settings
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Tap on the burger menu > select Schedule 
4. Attempt to select the "add to calendar" buttons next to events 
5. Note that buttons cannot be selected 

Browser/Device: 
Chrome - Google Nexus 6 / Android 5.1.1

![gio_673](https://cloud.githubusercontent.com/assets/15128124/14472147/9ee64cb8-00a6-11e6-877d-a5cfb84f6d62.png)
